### PR TITLE
its a pr

### PR DESCRIPTION
**Pull request by Commandserver**

### CODE REVIEW

File: docpages/INDEX.md [hunk 1]
```diff
@@ -45,10 +45,18 @@ You can find further releases in other architectures and formats or the source c
...
-The library may work fine in other operating systems too, if you run a D++ bot on something not listed here please let us know!
+### Mac OS X and FreeBSD
+The library is well-functional and stable on **Mac OS X** and **FreeBSD** too.
+For running your bot on a **Raspberry Pi**, we offer a prebuilt .deb package for ARM64, ARM6, and ARM7 so that you do not have to wait for it to compile.
+
+### Windows
+**Windows** is well-supported with ready-made compiled DLL and LIB files, please check out our [Windows Bot Template repository](https://github.com/brainboxdotcc/windows-bot-template). The Windows Bot repository can be cloned and integrated immediately into any Visual Studio 2019 and 2022 project in a matter of minutes.
+
+### Other OS
+The library should work fine on other operating systems as well, and if you run a D++ bot on something not listed here, please let us know!
```
Comment (Commandserver) [inline docpages/INDEX.md:59]: copied from the README.md...

File: include/dpp/guild.h [hunk 2]
```diff
@@ -492,17 +516,17 @@ class DPP_EXPORT guild : public managed, public json_interface<guild>  {
...
 
-	/** Maximum users in a video channel, or 0 */
-	uint16_t max_video_channel_users;
+	/** Voice AFK timeout in seconds before moving users to AFK channel. Can be set to: 60, 300, 900, 1800, 3600 */
+	uint16_t afk_timeout;
 
-	/** Voice AFK timeout before moving users to AFK channel */
-	uint8_t afk_timeout;
+	/** Maximum users in a video channel, or 0 */
+	uint8_t max_video_channel_users;
```
Comment (Commandserver) [inline include/dpp/guild.h:523]: the afk_timeout field is now uint16_t & max_video_channel_users is now uint8_t, just because it's enough

**Review by Commandserver — COMMENTED** [review]
ready

File: docpages/INDEX.md [hunk 3]
```diff
@@ -45,10 +45,18 @@ You can find further releases in other architectures and formats or the source c
...
 
-The library runs great on **Linux**. **Windows** is also supported and we offer ready made compiled DLL and LIB files for easy integration into any windows visual studio 2019/2022 project.
-**Mac OS X** and **FreeBSD** is also functional and stable, as is running your bot on a **Raspberry Pi** — we offer a prebuilt .deb for ARM64, ARMv6 and ARMv7 to save on having to wait for it to compile.
+### Linux
+The library runs ideally on **Linux**.
 
-The library may work fine in other operating systems too, if you run a D++ bot on something not listed here please let us know!
+### Mac OS X and FreeBSD
+The library is well-functional and stable on **Mac OS X** and **FreeBSD** too.
+For running your bot on a **Raspberry Pi**, we offer a prebuilt .deb package for ARM64, ARM6, and ARM7 so that you do not have to wait for it to compile.
```
Comment (craigpipeandpiper) [inline docpages/INDEX.md:53]: pi/arm needs its own section

[the remark this answers is not in the document]
Comment (Commandserver) [reply docpages/INDEX.md:53]: should i also change that in the readme?

[the remark this answers is not in the document]
Comment (Commandserver) [reply docpages/INDEX.md]: done

**Pull request merged**